plugins/permission/api-report.md

```diff
@@ -0,0 +1,48 @@
+## API Report File for "@backstage/plugin-permission"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Permission } from '@backstage/plugin-permission-common';
+import { default as React_2 } from 'react';
+import { RouteProps } from 'react-router';
+
+// @public (undocumented)
+export type AsyncPermissionResult = {
+  loading: boolean;
+  allowed: boolean;
+  error?: Error;
+};
+
+// Warning: (ae-unresolved-link) The @link reference could not be resolved: The package "@backstage/plugin-permission" does not have an export "NotFoundErrorPage"
```

Comment:
Is there a way to link to something not from this package?

Reply:
Yep! Here's an example: https://github.com/backstage/backstage/blob/master/packages/app-defaults/src/defaults/components.tsx#L76